django_declarative_apis/machinery/__init__.py

```diff
@@ -907,6 +907,11 @@ class ResourceEndpointDefinition(EndpointDefinition):
     :code:`ResourceEndpointDefinition` is mainly used for :code:`GET`.
     """
 
+    def __init__(self, *args, **kwargs):
```

Comment:
There's already an `__init__` further down - I assume we should merge these @dshafer ?